sbt-cross/src/main/scala/sbtcross/SbtCrossPlugin.scala

```diff
@@ -0,0 +1,31 @@
+package sbtcross
+
+import sbt._
+import StringUtilities.nonEmpty
+
+import scala.language.implicitConversions
+
+object SbtCrossPlugin extends AutoPlugin {
+  val autoImport = AutoImport
+  object AutoImport extends JvmCross with CrossKeys {
+    final implicit def toScalaJSGroupID(groupID: String) = null
+
+    final implicit def toCrossGroupID(groupID: String): CrossGroupID = {
+      nonEmpty(groupID, "Group ID")
+      new CrossGroupID(groupID)
+    }
+
+    final implicit def crossProjectFromBuilder2(
+        builder: CrossProject.Builder): CrossProject = {
+      builder.crossType(CrossType.Full)
+    }
+
+    final implicit def crossClasspathDependencyConstructor2(
+        cp: CrossProject): CrossClasspathDependency.Constructor =
+      new CrossClasspathDependency.Constructor(cp)
+
+    final implicit def crossClasspathDependency2(
+        cp: CrossProject): CrossClasspathDependency =
+      new CrossClasspathDependency(cp, None)
```

Comment:
These two implicit conversions could be put in the companion object of `CrossProject`, and they wouldn't need the awful `2`.